### PR TITLE
Migrate @storybook/addon-graphql to typescript

### PR DESCRIPTION
**Pull request by lonyele**
Issue: #5030 

## What I did

- I migrated `@storybook/addon-graphql` to typescript. 
- Remove duplicates and change some codes to named export/import and arrow function

I asked discord's typescript channel about the contribution. It seems like no one has been working on this addon so I took this one. It is not on the priority list but it looks like a good starter for me. It is my first time migrating to typescript here, so please give me some feedbacks. I'll apply it.

### CODE REVIEW

File: addons/graphql/src/components/FullScreen/style.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-export default {
+export const style = {
   wrapper: {
-    position: 'absolute',
+    position: 'absolute' as 'absolute',
```
Comment (gaetanmaisse) [inline addons/graphql/src/components/FullScreen/style.ts]: You can use `const` assertions, see [TypeScript 3.4 Release Note](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html): `position: 'absolute' as const,`

File: addons/graphql/src/shared.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    const body = JSON.stringify(params);
+    const options = Object.assign({ body }, FETCH_OPTIONS);
+    return fetch(url, options).then((res: any) => res.json());
+  };
+};
+
+export const reIndentQuery = (query: string) => {
+  const lines = query.split('\n');
+  const spaces = lines[lines.length - 1].length - 1;
+  return lines.map((l: string, i: number) => (i === 0 ? l : l.slice(spaces))).join('\n');
```
Comment (gaetanmaisse) [inline addons/graphql/src/shared.ts]: I think types of `l` and `i` will be automatically inferred by TS as soon as you typed `query`, so maybe they are useless

**Review by gaetanmaisse — APPROVED** [review]

**Comment by gaetanmaisse** [comment]
@lonyele great PR, well split in multiple commits and so easy to review, I just added a few minors comments. 👏

**Review by ndelangen — APPROVED** [review]

File: addons/graphql/src/components/FullScreen/style.ts (hunk 1 above)
Comment (lonyele) [reply addons/graphql/src/components/FullScreen/style.ts]: I saw that on other migrations here(yours) and didn't catch lastest typescript and applied what I alreay knew... shame on me. I applied it!

File: addons/graphql/src/shared.ts (hunk 2 above)
Comment (lonyele) [reply addons/graphql/src/shared.ts]: Yeap, It is inferred. I should be cautious about next time.

**Review by gaetanmaisse — APPROVED** [review]

**Comment by lonyele** [comment]
@gaetanmaisse Thanks for your review. I hopefully make better one next time.

**Comment by ndelangen** [comment]
FYI will merge this after 5.1 is released

**Pull request merged**